assets/operatingInfo/za.private.json

```diff
@@ -1,18 +1,18 @@
 {
   "timezone": "Africa/Johannesburg",
   "holidays": {
-    "01/01/2021":"New Years Day",
-    "03/22/2021":"Human Rights Day",
-    "04/02/2021":"Good Friday",
-    "04/05/2021":"Family Day",
-    "04/27/2021":"Freedom Day",
-    "05/01/2021":"Workers Day",
-    "06/16/2021":"Youth Day",
-    "08/09/2021":"National Womens Day",
-    "09/24/2021":"Heritage Day",
-    "12/16/2021":"Day of Reconciliation",
-    "12/25/2021":"Christmas",
-    "12/27/2021":"Boxing Day"
+    "01/02/2022":"New Years Day",
```

Comment:
Is the update New Year's day really Jan 2nd?

Reply:
> Is the updated New Year's day really Jan 2nd?

Thanks, yes for 2023.